### ISSUE TITLE
Make form post to relative URL

### ISSUE THREAD

**Issue by buske** [Genecloud/simplebeacon#1]
Instead of `/query`, if the post request is to `query` then the form template will still work even when served from a non-root endpoint (e.g. `mysite.org/api/beacon`)

**Comment by bartgrantham** [comment]
Good idea.  I'm probably going to move the beacon and OIDC endpoints into flask blueprints in a few merges.  Have a lot to clean up to get from here to there, so I'll do a quick edit to accomplish this for now.
